duckduckgo/iOS
# show forced encryption state

Task/Issue URL: https://app.asana.com/0/392891325557410/518282040737111
Tech Design URL:
CC:

**Description**:

Shows forced encryption state after upgrading a http request to https.

Have recreated TLDs logic in the app otherwise redirects made the app lose the forced state for a given site. As such the data in tlds.js is now in a json file read in by the app and injected in to the tlds.js script when loading content blocking files.

**Steps to test this PR**:

1. Go to http://booking.com 
1. After it finishes redirecting, check the privacy dashboard - should say Forced Encryption  on the overview and encryption detail screen

Other sample sites:

http://happyearth.marketing -> Unencrypted
http://falkirkrpg.org.uk -> Encrypted (redirects to https but not upgraded/forced because not in the list)
https://falkirkrpg.org.uk/noxwall/mixed.html -> Mixed

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
